webhook.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+
+curl -dH -X POST "$WEBHOOK_URL"
```

Comment:
One nice addition here is to also add the `--fail` flag - otherwise Azure returning an error code is still considered a success because curl is reporting that it successfully sent the request, regardless of the result. That can be a bit confusing, as your pipeline might report success even though it actually failed